database/migrations/2016_02_17_175810_create_competition_user_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCompetitionUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('competition_user', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('competition_id')->unsigned()->index();
+            $table->foreign('competition_id')->references('id')->on('competitions')->onDelete('cascade');
```

Comment:
@sbsmith86 I think it's more if you delete a competition user it will delete the reference in the pivot table for the competitions that user was associated with.